peripherals/flash/flash_wl.yaml

```
@@ -0,0 +1,269 @@
+FLASH:
+  ACR:
+    EMPTY:
+      Programmed: [0, "User Flash programmend"]
+      Empty: [1, "User Flash empty"]
+    PES:
+      Granted: [0, "Flash program and erase operations granted"]
+      Suspended: [1, "Any new Flash program and erase operation is suspended until this bit is cleared. The PESD bit in FLASH_SR is set when PES bit in FLASH_ACRis set"]
+    DCRST:
+      NotReset: [0, "Data cache is not reset"]
+      Reset: [1, "Data cache is reset"]
+    ICRST:
+      NotReset: [0, "Instruction cache is not reset"]
+      Reset: [1, "Instruction cache is reset"]
+    DCEN:
+      Disabled: [0, "Data cache is disabled"]
+      Enabled: [1, "Data cache is enabled"]
+    ICEN:
+      Disabled: [0, "Instruction cache is disabled"]
+      Enabled: [1, "Instruction cache is enabled"]
+    PRFTEN:
+      Disabled: [0, "Prefetch is disabled"]
+      Enabled: [1, "Prefetch is enabled"]
+    LATENCY:
+      WS0: [0, "0 wait states"]
+      WS1: [1, "1 wait states"]
+      WS2: [2, "2 wait states"]
+      WS3: [3, "3 wait states"]
+      WS4: [4, "4 wait states"]
+      WS5: [5, "5 wait states"]
+      WS6: [6, "6 wait states"]
+      WS7: [7, "7 wait states"]
+      WS8: [8, "8 wait states"]
+      WS9: [9, "9 wait states"]
+      WS10: [10, "10 wait states"]
+      WS11: [11, "11 wait states"]
+      WS12: [12, "12 wait states"]
+      WS13: [13, "13 wait states"]
+      WS14: [14, "14 wait states"]
+      WS15: [15, "15 wait states"]
```

There are only 3 wait states:
```suggestion
      WS0: [0, "0 wait states"]
      WS1: [1, "1 wait states"]
      WS2: [2, "2 wait states"]
```